src/apify_client/clients/resource_clients/webhook.py

```diff
@@ -0,0 +1,130 @@
+from typing import Any, Dict, List, Optional
+
+from ..base.resource_client import ResourceClient
+from .webhook_dispatch_collection import WebhookDispatchCollectionClient
+
+
+def prepare_webhook_representation(
```

Comment:
I'd rename to `_prepare_webhook_representation` since this is just internal function.

Reply:
Good point, I am not used to the underscore-before-function-name notation.

Reply:
me neither, but seems we are using it so let's try to be consistent